src/SFA.DAS.EmployerAccounts.Jobs/StartupJobs/CreateReadStoreDatabaseJob.cs

```diff
@@ -28,25 +29,54 @@ public async Task Run()
                 Id = DocumentSettings.DatabaseName
             };
 
-            var documentCollection = new DocumentCollection
+            var documentCollections = new List<DocumentCollection>
```

Comment:
Will this job update the read store if it already exists?

Reply:
If you are talking about the scenario where the readstore exists with the existing table but not the new "AccountSignedAgreement" table then it will be fine as it does a create if not exists at the level of each table. One thing we need to consider is migrating existing signed agreements into this readstore when we deploy it for the first time.

Reply:
solution for migrating existing agreements seems to be a run once job. I will write this next.